tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -400,6 +400,7 @@ Restore PVC Value To Default Size
     Click Button    Restore Default
     Wait Until Keyword Succeeds    30    1
     ...    Wait Until Page Contains    Cluster settings updated successfully.
+    Sleep    120s
```

Comment:
Could you add a reason for the Sleep? 

e.g.  ` Sleep    120s    reason=Wait until configuration is applied and traefik-proxy is restored`

Reply:
Done